daisy_workflows/build-publish/windows/windows-server-2012-r2-dc-bios.publish.json

```diff
@@ -0,0 +1,47 @@
+{{/*
+  Template to publish Windows server UEFI images.
+  By default this template is setup to publish to the 'gce-image-builder'
+  project, the 'environment' variable can be used to publish to 'test', 'prod',
+  or 'staging'.
+  DeleteAfter is set to 180 days for all environments other than prod where no
+  time period is set.
+*/}}
+{
+  "Name": "windows-server-2012-r2-dc-bios",
+  {{$work_project := printf "%q" "gce-image-builder" -}}
+  {{$endpoint := `"https://www.googleapis.com/compute/alpha/projects/"` -}}
+  {{$delete_after := `"24h*30*6"` -}}
+  {{if eq .environment "prod" -}}
```

Comment:
For all of these publish files, I think they should only be able to publish to `bct-prod-images`

Reply:
Follow up: Is the byol license the right one here? I figure we don't want to be using the billed ones for testing?

Reply:
Discussed offline to use payg instead of byol